helm/templates/deployment-vllm-multi.yaml

```diff
@@ -153,6 +153,12 @@ spec:
           - name: LMCACHE_REMOTE_SERDE
             value: "{{ .Values.cacheserverSpec.serde }}"
           {{-   end }}
+          {{-   if and .Values.lookupserverSpec $modelSpec.lmcacheConfig.p2pEnabled}}
+          - name: LMCACHE_LOOKUP_URL
+            value: "{{ include "lookupserver.formatLookupUrl" (dict "service_name" (print .Release.Name "-lookup-server-service") "port" .Values.lookupserverSpec.servicePort) }}"
```

Comment:
This wouldn't resolve to the correct usable service address, instead it should be something like:
```- name: LMCACHE_LOOKUP_URL
  value: "{{ printf \"%s.%s.svc.cluster.local:%d\" (print .Release.Name \"-lookup-server-service\") .Release.Namespace .Values.lookupserverSpec.servicePort }}"````